eogrow/core/pipeline.py

```diff
@@ -222,7 +226,7 @@ def run(self) -> None:
                 pipeline_timestamp=timestamp,
             )
 
-            LOGGER.info("Running %s", self.__class__.__name__)
+            LOGGER.info("Running %s", self._pipeline_name)
```

Comment:
I would still keep the original line, and rather print the custom name additionally in case of providing specifics. I think the class name does bring value already on its own.

perhaps the `custom_name` is not a suitable name of the parameter, since the user would in such cases want to provide extra information about the pipeline? 

As mentioned in the example internally, in case of the following:

```
[
  {"**DEM": "${config_path}/01_download_DEM.json"},
  {"**RABA": "${config_path}/02_rasterize_RABA.json"},
  {"**mowing": "${config_path}/02_rasterize_mowing_parcels_2019.json"},
  {"**GERK": "${config_path}/02_rasterize_GERK_SEGMENT_2019.json"},
  {"**travniki": "${config_path}/02_rasterize_travniki.json"},
  {"**convert": "${config_path}/03_convert.json"}
]
```

the pipelines from the second to second-last are all rasterization pipelines, and I would like to know they are rasterization pipelines, but additionally also would like to have the option to specify the details of what kind of rasterization I'm doing, so that it's easier to find the logs and interpret the logs

we can also have a discussion if that would be easier

Reply:
discusse in private, resolving